source-code/cli/src/commands/machine/translate.ts

```diff
@@ -1,7 +1,7 @@
 // @ts-nocheck
 import { query } from "@inlang/core/query"
 import { Command } from "commander"
-import { countMessagesPerLanguage, getFlag, log } from "../../utilities.js"
+import { getFlag, log } from "../../utilities.js"
```

Comment:
please make sure that the build runs after refactoring code and before you push your code! I have disabled the `build` command for the CLI for now

Reply:
sorry, moved old code before i ran off to dinner